packages/core/src/pill-input/PillInput.css

```diff
@@ -258,11 +258,11 @@
 }
 
 .saltPillInput-bordered > .saltPillInput-endAdornmentContainer {
-  padding-top: calc(var(--salt-spacing-50) - var(--salt-size-border));
+  padding-top: calc(var(--salt-spacing-50) - var(--salt-size-fixed-100));
```

Comment:
as this is padding should it be spacing-fixed instead of size-fixed?

Reply:
This calculation is referring to the thickness of the border, but happy to change it